pytensor/tensor/basic.py

```diff
@@ -1925,30 +1925,18 @@ def make_node(self, x, axis, splits):
     def perform(self, node, inputs, outputs):
         x, axis, splits = inputs
```

Comment:
Is variable axis a frequent use case? If not why not setting it in the `Op.__props__`?

Reply:
It's as supported as variable axis in `Join`, for tho grad of which `Split` is used.

Reply:
This shouldn't pose any problem when users use constant axis